_posts/blog/2019-02-28-mlab-announcement.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: blog
+title: "After A Decade of Growth, Measurement Lab Spins Out of New America to Join Code for Science & Society"
+author: "Chris Ritzo"
```

Comment:
Can we list multiple authors? maybe this should come from both of us? Or maybe the author should be "M-Lab Team" since it's also the CS&S and OTI folks?

Reply:
Either is possible.